mvx/caip10.md

```diff
@@ -0,0 +1,59 @@
+---
+namespace-identifier: mvx-caip10
+title: MultiversX Namespace - Addresses
+author: MultiversX Team <contact@multiversx.com>
+discussions-to: https://github.com/ChainAgnostic/namespaces/issues/89
+status: Draft
+type: Standard
+created: 2023-09-11
+requires: ["CAIP-2", "CAIP-10"]
+---
+
+# CAIP-10
+
+_For context, see the [CAIP-10][] specification._
+
+## Rationale
+
+The MultiversX Address format is bech32, specified by the [BIP_0173][].
+The human-readable-part of the bech32 addresses is "erd" which means that the MultiversX address always starts with an `erd1` ("1" is a separator) e.g.: `erd1sea63y47u569ns3x5mqjf4vnygn9whkk7p6ry4rfpqyd6rd5addqyd9lf2`.
+
+The MultiversX network defines the address of an account as the bech32 representation of the public key of its corresponding pair of keys (the secret key remains known only to the user that owns the key pair). The public key is 32 bytes in length (64 hex characters), while the address (bech32) is 62 characters long.
```

Comment:
I assume this byte-->char conversion is base58btc encoding? Check out other CAIP-10 profiles for examples of how people define this dependency normatively.

Reply:
No, the byte->char conversion is just the direct transformation from the public key bytes to the bech32 string representation: [https://en.bitcoin.it/wiki/Bech32](https://en.bitcoin.it/wiki/Bech32)

Reply:
whoops my mistake had bech32 mixed up with something else in my mind 😅